wiki/Gameplay/Judgement/osu!/en.md

```diff
@@ -1,41 +1,58 @@
+| ![](/wiki/shared/judgement/osu!/hit50.png "50") | 50 | The third best achievable result for a hit object, the last one before a miss. Usually denoted with an orange colour. Scoring a 50 will prevent the appearance of a Katu or a Geki at the combo's end. |
+| ![](/wiki/shared/judgement/osu!/hit0.png "Miss") | Miss | The worst possible result for a hit object. Scoring a Miss will reset the current combo to 0 and will prevent the appearance of a Katu or a Geki at the combo's end. |
 
 ## Judgement mechanics
 
-**Hit circles:**
+### Hit circles
+
+- A hit circle is judged as a 300, 100, or 50 depending on how accurate the player's timing is when clicking on it.
+  The time intervals used to determine the concrete result for a hit circle are called *hit windows*, and are influenced by the beatmap's [overall difficulty](/wiki/Beatmap/Overall_difficulty#osu!).
+- A hit circle is judged as missed when:
+  - The circle is hit before the start of the 50 hit window and does not trigger [notelock](/wiki/Gameplay/Judgement/Notelock).
+  - The circle is hit after the end of the 50 hit window, or not hit at all.
+
+### Sliders
+
+Sliders consist of parts such as the slider head, slider tail, slider ticks, and slider repeats. The slider as a whole is judged based on how many parts of the slider the player has hit, as outlined below:
+
+- 300: The player successfully hit all slider parts.
+- 100: The player hit at least half of the slider parts.
+- 50: The player managed to hit at least one of the parts.
+- Miss: The player did not successfully hit any of the slider parts.
+- If [ScoreV2](/wiki/Gameplay/Game_modifier/ScoreV2) is active, the accuracy of hitting the slider head is additionally accounted for when judging the entire slider:
+  - Getting a 300 for a slider requires getting a 300 judgement for the slider head.
+  - Getting a 100 for a slider requires getting a 100 judgement or better for the slider head.
+
+The slider parts also have additional influence on the current [combo](/wiki/Gameplay/Combo_(score_multiplier)):
+
+  - Tapping the slider head too early does not incur a miss, but will reset combo to 0.
+  - Missing a slider tick or repeat does not incur a miss, but will reset combo to 0.
+  - Missing the slider end does not incur a miss, but will not increment combo.
+
+### Spinners
+
+Each spinner has a set number of spins required to complete it. This number depends on the [overall difficulty](/wiki/Beatmap/Overall_difficulty#sliders-and-spinners) of the beatmap. The criteria for judging the spinner as a whole are as follows:
 
-- A 300, 100 or 50 given from a normal hit circle depending on how accurately the player tap on it.
-- A miss given if the player do not click on the hit circle at all or click on it way too early.
+- 300: The player successfully performed the number of spins required for completion, or more.
+- 100: The player performed one spin less than the required number.
```

Comment:
```suggestion
- 100: The player performed fewer spins than the required number.
```

Reply:
This suggestion loses information. The point was that there is a one-spin difference between minimum spins for a 300 and minimum spins for a 100.

Reply:
?

![image](https://user-images.githubusercontent.com/36758269/174487311-5376966f-7e3c-4c5e-9af1-1859c06ec802.png)

Reply:
That's correct.

Reply:
ok lol